content/other/phase-angles/inverse/meta.json

```
@@ -0,0 +1,27 @@
+{
+  "abstract": null,
+  "authors": [
+    {
+      "name": "Guillermo Alonso"
```

Could we use Guillermo's PennyLane profile here instead?
```suggestion
      "username": "KetPuntoG"
```

---

We should be able to use both. Will update.